src/ansys/pre_commit_hooks/templates/README.md

```diff
@@ -0,0 +1,71 @@
+<!-- -->
+<a name="readme-top"></a>
+<!--
+*** {{ doc_repo_name }} README
+-->
+
+<p style="text-align: center;">
+    <br> English | <a href="README_CN.md">中文</a>
+</p>
+
+[![PyAnsys](https://img.shields.io/badge/Py-Ansys-ffc107.svg?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAIAAACQkWg2AAABDklEQVQ4jWNgoDfg5mD8vE7q/3bpVyskbW0sMRUwofHD7Dh5OBkZGBgW7/3W2tZpa2tLQEOyOzeEsfumlK2tbVpaGj4N6jIs1lpsDAwMJ278sveMY2BgCA0NFRISwqkhyQ1q/Nyd3zg4OBgYGNjZ2ePi4rB5loGBhZnhxTLJ/9ulv26Q4uVk1NXV/f///////69du4Zdg78lx//t0v+3S88rFISInD59GqIH2esIJ8G9O2/XVwhjzpw5EAam1xkkBJn/bJX+v1365hxxuCAfH9+3b9/+////48cPuNehNsS7cDEzMTAwMMzb+Q2u4dOnT2vWrMHu9ZtzxP9vl/69RVpCkBlZ3N7enoDXBwEAAA+YYitOilMVAAAAAElFTkSuQmCC)](https://docs.pyansys.com/)
+[![Python](https://img.shields.io/pypi/pyversions/{{ project_name }}?logo=pypi)](https://pypi.org/project/{{ project_name }}/)
+[![PyPI](https://img.shields.io/pypi/v/{{ project_name }}.svg?logo=python&logoColor=white)](https://pypi.org/project/{{ project_name }})
+[![Codecov](https://codecov.io/gh/ansys/{{ doc_repo_name }}/branch/main/graph/badge.svg)](https://codecov.io/gh/ansys/{{ doc_repo_name }})
+[![GH-CI](https://github.com/ansys/{{ doc_repo_name }}/actions/workflows/ci_cd.yml/badge.svg)](https://github.com/ansys/{{ doc_repo_name }}/actions/workflows/ci_cd.yml)
+[![MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![Black](https://img.shields.io/badge/code%20style-black-000000.svg?style=flat)](https://github.com/psf/black)
+
+## Overview
+
+A Python wrapper for Ansys {{ product }}.
+
+## Installation
+
+You can use `pip <https://pypi.org/project/pip/>`_ to install {{ doc_repo_name }}.
+
+```sh
+  pip install {{ project_name }}
+```
+
+To install the latest development version, run these commands:
+
+```sh
+  git clone {{ repository_url }}
+  cd {{ doc_repo_name }}
+  pip install -e .
+```
+
+For more information, see [Getting Started](https://{{ product }}.docs.pyansys.com/version/stable/getting_started/index.html).
+
+## Basic usage
+
+This code shows how to import {{ doc_repo_name }} and use some basic capabilities:
+
+``` python
+    print("Put sample code here")
+```
+
+For comprehensive usage information, see [Examples](https://{{ product }}.docs.pyansys.com/version/stable/examples.html) in the [{{ doc_repo_name }} documentation](https://{{ product }}.docs.pyansys.com/version/stable/index.html).
+
+## Documentation and issues
+
+Documentation for the latest stable release of {{ doc_repo_name }} is hosted at [{{ doc_repo_name }} documentation](https://{{ product }}.docs.pyansys.com/version/stable/index.html).
+
+In the upper right corner of the documentation's title bar, there is an option for switching from
+viewing the documentation for the latest stable release to viewing the documentation for the
+development version or previously released versions.
```

Comment:
I would avoid documenting this. Anyway, documentation team has the last word on this.